.github/workflows/main.yml

```diff
@@ -136,49 +95,35 @@ jobs:
           echo $HOME
           echo $DISPLAY
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: true
       - run: sudo apt-get update -y && sudo apt-get install -y libusb-1.0.0 libudev-dev
       - name: Install rust
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y cmake binutils-dev libcurl4-openssl-dev libiberty-dev libelf-dev libdw-dev
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path --default-toolchain none -y;
-      - name: Install node
-        uses: actions/setup-node@v2
+        uses: actions-rs/toolchain@v1
         with:
-          node-version: "16"
+          toolchain: stable
+      - name: Install node
+        uses: actions/setup-node@v3
       - name: Install yarn
         run: |
-          npm install -g yarn
-      - name: Build Ledger app
-        run: |
-          make
-      - name: Build/Install build js deps
+          npm install -g yarn ts-node
+      - name: Build and run zemu tests
         run: |
-          export PATH=~/.cargo/bin:$PATH
-          make zemu_install
-      - name: Run zemu tests
-        run: |
-          export PATH=~/.cargo/bin:$PATH
-          make zemu_test
+          make test_all
 
   build_package_nanos:
-    needs:
-      [configure, build, build_nano_S, build_nano_X, build_nano_SP, test_zemu]
+    needs: [configure, build, build_ledger, test_zemu]
```

Comment:
I think that build package steps should look like this: https://github.com/Zondax/ledger-arweave/blob/a18dd3494b00cf0ee762cdd19b00f05fd66a95b1/.github/workflows/main.yml#L90-L160
We don't have light and XL version in Stacks